facade/nodeFacade.go

```diff
@@ -287,6 +287,11 @@ func (nf *nodeFacade) GetAllESDTTokens(address string) (map[string]*esdt.ESDigit
 	return nf.node.GetAllESDTTokens(address)
 }
 
+// GetAllIssuedESDTs returns all the issued esdts for a given address
```

Comment:
`// GetAllIssuedESDTs returns all the issued esdts from the esdt system smart contract`

Reply:
done